src/watchd/runtime.rs

```diff
@@ -153,6 +166,9 @@ impl Runtime {
                     if *required_height >= tx_status.block_pos.map(|b| b.pos).unwrap_or_default() {
                         let service_id = service_id.clone();
                         self.untrack(tx_status.txid);
+                        self.electrum_worker
+                            .untrack_transaction(tx_status.txid)
+                            .expect("unable untrack transaction in electrum worker");
```

Comment:
why we panic on error here instead of propagating it up? This means that any temporary network connectivity issue (like Electrum server having too many connections) will kill the daemon thread and this would require manual node restart...